R/insertions.R

```diff
@@ -205,7 +214,7 @@ setMethod("func", "Heading", function (x) return(NA))
 #' @rdname Insertions
 #' @export
 setMethod("anchors", "Insertions", function (x) {
-    f <- vapply(x, anchor, integer(1))
+    f <- vapply(x, anchor, 1)
```

Comment:
This line looks odd.

Reply:
Oph, yeah. Since `anchor()` could now return either an integer or a character I've switched this to `unlist(lapply())`